pkgs/http/lib/src/client.dart

```diff
@@ -198,6 +198,10 @@ Client? get zoneClient {
 /// and the convenience HTTP functions (e.g. [http.get]). If [clientFactory]
 /// returns `Client()` then the default [Client] is used.
 ///
+/// Only fresh `Client` instances using the default constructor are impacted.
+/// HTTP requests made using `dart:io` or `dart:html` APIs,
+/// or using specifically instantiated client implementations, are not affected.
+///
 /// When used in the context of Flutter, [runWithClient] should be called before
```

Comment:
I was going to replace the Flutter section. Are you working on that or should I keep going?

Reply:
> Are you working on that or should I keep going?

I wasn't planning on looking more deeply at the Flutter stuff. I noticed this was a gap in the docs, even outside of the specifics of Flutter usage, so I think getting the general message added is a useful and easy first step.